src/Microsoft.AspNet.Mvc.ModelBinding/Metadata/BindingMetadata.cs

```diff
@@ -29,6 +29,13 @@ public class BindingMetadata
         public Type BinderType { get; set; }
 
         /// <summary>
+        /// Gets or sets a value indicating whether or not the property can be model bound.
+        /// Will be ignored if the model metadata being created is not a property.
```

Comment:
changed it to `Will be ignored if the model metadata being created does not represent a property`